documentation/adr/003-target-TSE-SE-AWS-reference-architecture.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: page.njk
+status: Proposed
+date: git Created
+tags:
+  - adr
+order: 3
+title: 003 - Target the TSE-SE Reference Architecture 
+---
+
+**Date: {{ page.date.toISOString().substring(0,10) }}**
+
+**Status: {{ status }}**
+
+## Context
+During a review of the architecture inherited from the outgoing supplier, we questioned choices regarding account structure, networking, IPAM and the tenant concept. These concerns were addressed in subsequent iterations of the target architecture and shared with AWS for review.
```

Comment:
I wouldn't even mention this. They were doing Discovery work and was never a serious proposition for production

Reply:
The ADR isn't agreeing whether they did a good job or not (or showing they did a bad one)

Reply:
I agree with this - we should focus more on the communication with AWS (which is below) and providing the context of why that is suitable. I think the  next 2 paragraphs pretty much cover it.